### ISSUE TITLE
[Feature request]: More flexibility for code-bubble-controls

### ISSUE THREAD

**Issue by isabelacmor** [break-stuff/code-bubble#3]
I can foresee it being useful for users to add custom controls to manipulate the preview area. I imagine this would be configurable via the `CodeBubbleConfig['component']` object, so its set at a global level rather than a per-`code-bubble` level?

One example we're hoping to leverage this feature for is a theme toggler. In our case, this would be a dropdown with some theme options. On selecting an option, our app would handle updating whatever theming / classnames updates are needed to provide the right styles to the component being previewed.

**Comment by break-stuff** [comment]
I don't think we will be adding custom controls into the component at this time, but you should be able to get the external controls to work with the controls through either the hooks or `updateConfig` method.
